web/src/routes/(user)/people/[personId]/+page.svelte

```diff
@@ -57,12 +58,34 @@
   let people = data.people.people;
   let personMerge1: PersonResponseDto;
   let personMerge2: PersonResponseDto;
+  let potentialMergePeople: PersonResponseDto[] = [];
 
   let personName = '';
 
+  let name: string;
+  let suggestedPeople: PersonResponseDto[] = [];
+
   $: isAllArchive = Array.from($selectedAssets).every((asset) => asset.isArchived);
   $: isAllFavorite = Array.from($selectedAssets).every((asset) => asset.isFavorite);
 
+  $: {
+    if (name != '') {
+      suggestedPeople = [];
+      for (const person of people) {
+        if (person.name.startsWith(name) && person.id != data.person.id) {
+          suggestedPeople.push(person);
+
+          // Limit the size of filteredArray to 5
+          if (suggestedPeople.length === 5) {
+            break;
+          }
+        }
+      }
+    } else {
+      suggestedPeople = [];
+    }
+  }
```

Comment:
Is there any reason you can't turn this into something like `$: suggestedPeople = !name? [] : people.filter(...).slice(5)`

Reply:
Not really, it was for readability during my tests